src/memray/_memray.pyx

```diff
@@ -855,8 +856,9 @@ cdef class FileReader:
     cdef HighWatermark _high_watermark
     cdef object _header
     cdef bool _report_progress
+    cdef int _memory_snapshot_stride
```

Comment:
I wonder if we should make this an `ssize_t` instead of an `int`... it probably doesn't matter too much (a program would need to run for more than 6 months before the number of generated memory snapshots was bigger than int max), but it'd be slightly more correct.

Reply:
Yeah, I don't mind to settle with size_t